openedx/features/partners/tests/test_reset_password.py

```diff
@@ -0,0 +1,59 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
+
+    def setUp(self):
+        self.partner_reset_password_end_point = reverse('partner_reset_password')
+        self.user = UserFactory(username='testuser', password='12345', email='abc@test.com')
+
+
+    def test_valid_partner_email(self):
+        """
+                Providing correct email
```

Comment:
We usually don't indent the docstring: Please change this like:
`"""`
`Providing correct email`
`"""`
here and at all the other places.